ci-scripts/rhdh-setup/template/backstage/chart-values.yaml

```diff
@@ -84,6 +84,37 @@ upstream:
         value: "warn"
       - name: NODE_TLS_REJECT_UNAUTHORIZED
         value: "0"
+    extraContainers:
```

Comment:
@yogananth-subramanian Please update this also in the https://github.com/redhat-performance/backstage-performance/blob/main/ci-scripts/rhdh-setup/template/backstage/chart-values.image-override.yaml thx

Reply:
My bad, I missed that. Thank you for pointing it out.